css/moonstone-variables-light.less

```diff
@@ -95,6 +95,8 @@
 @moon-input-decorator-bg-color: white;
 
 @moon-icon-font-color: @moon-dark-gray;
+@moon-icon-font-disabled-color: rgba(187, 187, 187, 0.6);
+@moon-icon-bg-disabled-color: rgba(77, 77, 77, 0.2);
 @moon-icon-button-font-color: #999;
```

Comment:
Hi @Djspaceg 
That's not a proper value for light theme.
To prevent build fail, I put same value with dark theme but we need proper value for light theme.
Could you advise it?